docs/meshstack.openstack.index.md

```diff
@@ -48,3 +48,8 @@ Project Users get the following OpenStack roles:
 The actual access rights associated with these roles are managed by the OpenStack instance and are not part of meshStack.
 
 More details about the User Federation with OpenStack can be found [here](meshstack.openstack.index.md).
+
+## Federated Idp
+
+meshIdB is not required for the integration anymore. You can directly integrate your company-wide IdP to OpenStack. meshStack will make sure that users have access to the OpenStack projects they are assigned to in meshStack.
```

Comment:
d: I think still relevant information is missing. What are the things our customers have to consider to make this work? Explain the configuration needed in meshStack. And what needs to be done in OpenStack to actually match the users? This is at least configure the mapping to write users to the MeshUsers domain. Also explain here that in general all these users are put into that domain.

And provide a hint that the OpenStack screens in meshStack are not available in that mode.

Please think about everything that is needed for delivery or our customer to set up this feature successfully.

You should also explain that we create users if they don't exist and pick up existing ones if they already logged in. But only under the condition of them being present in the MeshUsers domain.

Reply:
The part that users will be created if they not exists describes the last line.
I'll integrate your other points.